### ISSUE TITLE
Balance, Crops for seed oil.

### ISSUE THREAD

**Issue by moller21** [GTNewHorizons/NewHorizons#5862]
Hey,

I wanted to test the peanuts from harvestcraft vs. rape, mostly because rape seemed very slow for all the effort you had to put into it. So i made some 11x11-13 plots for testing:

![2020-04-03_19 07 56](https://user-images.githubusercontent.com/42100910/78386787-81420b00-75de-11ea-9d71-d5c0939488f8.png)

After 30 minutes this is the result:

 1  | Items | Seed Oil
-- | -- | --
Peanut | 0 | 0 (Just grew now, about after 1 hour)
Peanut 4 sprinkler | 2700 | 97200
peanut 8 sprinkler(-4plot) | 4091 | 147276
Rapa 1-1 | 176 | 22000
Rape 10-10 | 379 | 47375
Rape 20-20 | 737 | 92125
Rape 31-31 | 1527 | 190875
Rape 23-31 | 1421 | 177625

So yeah, the difference isn't as big as i thought. The IC2 crops got alot better with higher stats, I guess this post is more about nerf sprinklers now? 

I still think the rapeseed could use a little buff, like setting the growth size to 2 after harvested so they'll only have to grow 2 times, or 33% faster. This again because they're so hard to obtain compared to peanuts.

But mostly posting cause I spent some time on it, and it shows the growth speed of crops in case you're curious.

I also made a spreadsheet to show the return after 30 mins to verify what i found, it's about the same. 

Gain | 1 | 3 | 5 | 7 | 9 | 11 | 13 | 15 | 17 | 19 | 21 | 23 | 25 | 27 | 29 | 31
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
Growth |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
1 | 160 | 169 | 178 | 188 | 198 | 208 | 220 | 231 | 244 | 257 | 271 | 285 | 300 | 316 | 333 | 351
3 | 202 | 213 | 224 | 236 | 249 | 262 | 276 | 291 | 307 | 323 | 340 | 359 | 378 | 398 | 419 | 441
5 | 243 | 256 | 269 | 284 | 299 | 315 | 332 | 350 | 369 | 388 | 409 | 431 | 454 | 478 | 503 | 530
7 | 283 | 298 | 314 | 331 | 349 | 368 | 387 | 408 | 430 | 453 | 477 | 502 | 529 | 557 | 587 | 618
9 | 323 | 340 | 358 | 377 | 398 | 419 | 441 | 465 | 490 | 516 | 544 | 572 | 603 | 635 | 669 | 704
11 | 362 | 381 | 402 | 423 | 446 | 470 | 495 | 521 | 549 | 578 | 609 | 642 | 676 | 712 | 749 | 789
13 | 400 | 422 | 445 | 468 | 493 | 520 | 548 | 577 | 608 | 640 | 674 | 710 | 747 | 787 | 829 | 872
15 | 439 | 462 | 487 | 513 | 540 | 569 | 599 | 631 | 665 | 700 | 738 | 777 | 818 | 861 | 907 | 955
17 | 476 | 502 | 528 | 557 | 586 | 618 | 651 | 685 | 722 | 760 | 800 | 843 | 887 | 934 | 984 | 1035
19 | 513 | 540 | 569 | 600 | 632 | 665 | 701 | 738 | 777 | 819 | 862 | 908 | 956 | 1006 | 1059 | 1115
21 | 549 | 579 | 610 | 642 | 676 | 712 | 750 | 790 | 832 | 876 | 923 | 971 | 1023 | 1077 | 1133 | 1193
23 | 585 | 616 | 649 | 684 | 720 | 759 | 799 | 841 | 886 | 933 | 982 | 1034 | 1089 | 1146 | 1206 | 1270
25 | 620 | 654 | 688 | 725 | 764 | 804 | 847 | 892 | 939 | 989 | 1041 | 1096 | 1153 | 1214 | 1278 | 1345
27 | 655 | 690 | 727 | 765 | 806 | 849 | 894 | 941 | 991 | 1043 | 1098 | 1156 | 1217 | 1281 | 1348 | 1419
29 | 689 | 726 | 765 | 805 | 848 | 893 | 940 | 990 | 1042 | 1097 | 1155 | 1216 | 1280 | 1347 | 1418 | 1492
31 | 723 | 761 | 802 | 844 | 889 | 936 | 986 | 1038 | 1092 | 1150 | 1211 | 1274 | 1341 | 1411 | 1485 | 1563

**Comment by sthegreat** [comment]
That would kind of defeat the point of the high quality oil as it would then be impossible for it to have the desired better ratios when processing with ethanol and/or NaOH.

**Comment by moller21** [comment]
I like the idea of gettin a more efficient recipe for bio diesel, not sure what i could be though. There's also the issue were you want regular seedoil for some things.

Canola has a bonus over peanuts when it makes more oil per craft, and I just noticed now it uses less power.

**Comment by Prometheus0000** [comment]
What about nerfing the sprinklers so that overlapping doesn't work? People would probably like that less, but it sounds more balanced to me. And it might help TPS issues on servers where people use a lot of them?

Also, could you test for what the difference is if they have bonemeal the whole time? Or the opposite if that's what you did.

**Comment by richardhendricks** [comment]
Preventing overlapping would computationally expensive. How would you "remember" which ones you have touched this tick? Anything we do to make them "worse" would just encourage more spam of them.  I don't recall any performance issues specifically tied to sprinklers (or ic2 crops in general for that matter). 

I dislike the idea of an intermediate fluid that has only one purpose, it is a cheap way to "improve" play IMO. NaOH is hardly a rate limiter IMO, getting ethanol/methanol in a reasonable way is a good limiter on how much seedoil you really need, IMO. 

I am also interested in if these numbers are with bonemeal or not. I'm guessing not?

**Comment by sthegreat** [comment]
I don't think it would be easily possible to prevent overlaps in general, sprinklers exclusively could probably be made to check their radius for other sprinklers every x seconds, but the problem is that there are several things that accelerate random ticks and they can of course synergize. The TC lamp thing and non-te WA immediately come to mind.

**Comment by moller21** [comment]
Yeah, this is without bonemeal.

**Comment by Prometheus0000** [comment]
So we should nerf sprinklers? To like 1/2 as often? 1/10? They do seem pretty powerful with basically 0 effort. It would be different if it actually required bonemeal.

Or give rape more oil?
